ivshmem/test/ivshmem-test.vcxproj

```diff
@@ -18,28 +18,25 @@
     <ProjectGuid>{87AEE822-FF8B-4743-A567-8B46F4FA6D99}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>ivshmemtest</RootNamespace>
-    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>$(LatestTargetPlatformVersion)</WindowsTargetPlatformVersion>
```

Comment:
Please add info about this patch into commit message. This is not clear why you update only one repository.

Reply:
Ok